### PR TITLE
Added config for more verbose errors on reset password

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
- Added settings.py docs
- Fixed left-overs from #4920

### CODE REVIEW

File: SETTINGS.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+`OPENSLIDES_USER_DATA_DIR`, but can be altered to another path.
+
+Email
+=====
+
+Please refer to the Django settings documentation. A changed email backend is
+usefull for debugging to print all email the the console::
+
+   EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
```
Comment (emanuelschuetze) [inline SETTINGS.rst]: here we should add our settings option: 
```DEFAULT_FROM_EMAIL = 'noreply@example.com'```

File: SETTINGS.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+SECURITY
+========
+
+For `DEBUG` and `SECRET_KEY` see the sections in the django settings
+documenataion.
+
+`RESET_PASSWORD_VERBOSE_ERRORS`: Default: `True`. Set to `False` to disable.
+Controls the verbosity on errors during a reset password. If enabled, an error
+will be shown, if there does not exist a user with a given email address. So one
+can check, if a email is registerd. If this is not wanted, disable verbose
```
Comment (emanuelschuetze) [inline SETTINGS.rst]: typo: register_e_d

File: SETTINGS.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+Directories
+===========
+
+`OPENSLIDES_USER_DATA_DIR`: The path, where all user data is saved, like static
+files, mediafiles and the default database. This path can be different to the
+location of the `settings.py`.
+
+`STATICFILES_DIRS` and `STATIC_ROOT`: Managing static files. Because the clint
+is not delivered by the serve anymore, these settings are obsolete.
```
Comment (emanuelschuetze) [inline SETTINGS.rst]: typo: serve_r_

File: SETTINGS.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+`MEDIA_ROOT`: The location of mediafiles. The default is a `media` folder insile
```
Comment (emanuelschuetze) [inline SETTINGS.rst]: typo: insi_d_e

File: SETTINGS.rst [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+`OPENSLIDES_USER_DATA_DIR`, but can be altered to another path.
+
+Email
+=====
+
+Please refer to the Django settings documentation. A changed email backend is
+usefull for debugging to print all email the the console::
```
Comment (emanuelschuetze) [inline SETTINGS.rst]: typo: useful

File: SETTINGS.rst [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+   EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
+
+Logging
+=======
+
+To setup basic logging see `logging
+<https://docs.djangoproject.com/en/2.2/topics/logging/>`_. 
+We recommend to enable all openslides related logging with level `INFO` per
```
Comment (emanuelschuetze) [inline SETTINGS.rst]: typo: OpenSlides

File: SETTINGS.rst [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+`PING_INTERVAL` and `PING_TIMEOUT` are settings for the clients how frequently
+to ping the server (interval) and how big is the timeout. If a ping took longer
+than the timeout, the clients does a forced reconnect.
+
+`COMPRESSION`: Enable or disables the compression when sending data. This does
+not affect the client.
+
+`PRIORITIZED_GROUP_IDS`: A list of group ids. If one client is logged in and the
+operator is in one of these groups, the client disconnectd and reconeects again.
```
Comment (emanuelschuetze) [inline SETTINGS.rst]: typo: disconnect_e_d and recon_n_ects

File: SETTINGS.rst (hunk 1 above)
Comment (FinnStutzenstein) [reply SETTINGS.rst]: This is not our own setting, but one of djangos pre-filled: https://docs.djangoproject.com/en/2.2/ref/settings/#default-from-email

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Somehow, one code change was gone.

I added a consistant, *translateable* error and response format: ```
{"detail": "some text with placeholder: {0}, {1}", "args": ["placeholder value", 42]}
```
The detail-part can be translated, if the error will be (frequently) seen by the user. In the most cases, errors should be rare, so no translation needed. But, for example, the error, that no user was found for an email during password reset, will be frequently seen. So the detail-part should be translated.

Can you add a translation for this error message? (see users/views.py)

File: SETTINGS.rst (hunk 1 above)
Comment (emanuelschuetze) [reply SETTINGS.rst]: Right.

File: SETTINGS.rst [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+<https://docs.djangoproject.com/en/2.2/ref/settings/>`_ to get an overview about
+all existing settings.
+
+SECURITY
+========
+
+For `DEBUG` and `SECRET_KEY` see the sections in the django settings
+documenataion.
+
+`RESET_PASSWORD_VERBOSE_ERRORS`: Default: `True`. Set to `False` to disable.
```
Comment (emanuelschuetze) [inline SETTINGS.rst]: The default value is not yet set in settings template...

**Comment by emanuelschuetze** [comment]
Rest looks good.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Done. Added link from the README (this was also gone somehow).

**Pull request merged**